### ISSUE TITLE
Do you have any result on other video dataset, like Charades?

### ISSUE THREAD

**Issue by visaVita** [SwinTransformer/Video-Swin-Transformer#29]
I finetuned the swin-base on Charades with the setting as follow:
1. optimizer:  I used AdamW with lr=75e-6, betas=(0.9,0.999), weight_decay=5e-2, other settings just follow the config that you provided.
2. learning policy:  CosineAnnealing with linear warmup by 2.5 epochs. 
3. loss function:  AsymmetricLoss [1] with neg=4 and pos=1
4. train_pipeline: clip_len=32, frame_intreval=2, num_clip=1, with RandomRescale (256,340) followed the setting in slowfast network, RandomResizedCrop, Resize(224,224) and Flip(0.5)
5. val_pipelne:  clip_len=32, frame_intreval=2, num_clip=10, Resize(-1, 256), CenterCrop(256), Flip(0.5)
When the total epoch is 30, I got final val map=44.96
When the total epoch is 60, I got final val map=45.88
Is my result correct? Do you have any suggestions about fine-tuning swin on other dataset?

ref: [1] Ben-Baruch, E., Ridnik, T., Zamir, N., Noy, A., Friedman, I., Protter, M., & Zelnik-Manor, L. (2020). Asymmetric loss for multi-label classification. arXiv preprint arXiv:2009.14119.
code: https://github.com/Alibaba-MIIL/ASL
